test/test_other.py

```diff
@@ -14248,3 +14248,16 @@ def test_fcvt(self):
 
   def test_llrint(self):
     self.do_other_test('test_llrint.c')
+
+  def test_strings(self):
+    self.do_other_test('test_strings.c', args=['wowie', 'too', '74'])
+
+  def test_strcmp_uni(self):
+    self.do_other_test('test_strcmp_uni.c')
+
+  def test_strndup(self):
+    self.do_other_test('test_strndup.c')
+
+  def test_errar(self):
```

Comment:
This is a pretty odd name.. not clear why you gave this name back in ea94dbe3b70f0772180463f412382d8abd02915a?

Reply:
IIRC the reason was that `test_error` triggered some CI response somewhere, because taken literally it says "test error"...